src/sys/aio.rs

```diff
@@ -86,7 +86,7 @@ pub struct AioCb<'a> {
     /// Could this `AioCb` potentially have any in-kernel state?
     in_progress: bool,
     /// Used to keep buffers from Drop'ing
-    keeper: Keeper<'a>
+    _keeper: Keeper<'a>
```

Comment:
These changes will be unnecessary after and will conflict with #820 .  You should revert the changes to this file.